src/components/avatar/UnknownUserAvatar.tsx

```diff
@@ -9,10 +10,10 @@ const UnknownUserAvatar = ({ className = '', height = 28, title, width = 28 }: S
         viewBox="0 0 28 28"
         width={width}
     >
-        <circle cx="14" cy="14" fill="#ededed" r="14" />
         <path
-            d="M5.5 25.1C6.7 21 10 18 14 18s7.3 3 8.5 7.1a14 14 0 0 1-17 0zM14 16a5 5 0 1 1 0-10 5 5 0 0 1 0 10z"
-            fill="#c3c3c3"
+            fill={vars.bdlGray50}
```

Comment:
Looks like this has WCAG contrast of 3.19, which should be ok for secondary elements.